src/TestSuite/Fixture/SchemaLoader.php

```diff
@@ -29,77 +31,79 @@
  * support multiple database platforms. You should use migrations
  * for that instead.
  */
-class SchemaManager
+class SchemaLoader
 {
+    use InstanceConfigTrait;
+
     /**
      * @var \Cake\Console\ConsoleIo
      */
     protected $io;
 
+    /**
+     * @var array
```

Comment:
we can now annotate the assoc arrays better with the next syntax IMO
```suggestion
     * @var array<string, mixed>
```

Reply:
you're right, we did switch to `array<>` for 4.3